oauth2_http/javatests/com/google/auth/oauth2/UserCredentialsTest.java

```diff
@@ -331,6 +339,34 @@ public void equals_false_refreshToken() throws IOException {
     assertFalse(otherCredentials.equals(credentials));
   }
 
+  @Test
+  public void equals_false_scopes() throws IOException {
```

Comment:
equalsFalseScopes per google style

Reply:
I kept the naming consistent with the rest of the file. I can update it, but it will be the only one like this. Or I can update the whole file if needed.

Reply:
Google style overrides pre-existing violations. You don't need to fix old violations, but please don't introduce new ones.